### PR TITLE
Quick setup

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

File: start/quick-setup.md [hunk 1]
```diff
@@ -21,10 +21,33 @@ When you develop with NativeScript, you can choose between a local tool set ([Na
...
+
+You may need to accept an User Account Control prompt to grant the script administrative privileges.
+
+ * On Mac:
+
+Using Spotlight, search `Terminal` and start it. This opens a console window. Copy and paste this script:
+
+> ruby -e "$(curl -fsSL https://github.com/NativeScript/nativescript-cli/blob/production/setup/native-script.rb)"
+
+The script calls some of the commands using `sudo` and you will need to provide your password several times.
```
Comment (rosen-vladimirov) [inline start/quick-setup.md]: that's strange, `sudo` is caching the "login" for some time, so you should be prompted only once

Comment (rosen-vladimirov) [inline start/quick-setup.md]: but I agree to keep it this way, just to be sure users are warned :)

**Comment by rosen-vladimirov** [comment]
:+1:

File: start/quick-setup.md (hunk 1 above)
Comment (teobugslayer) [inline start/quick-setup.md]: Some commands execute so slowly that the sudo authentication cache expires. See [the docs](http://linux.die.net/man/5/sudoers). At any rate, it _does_ prompt multiple times. But I will change the wording to "may".

File: start/quick-setup.md [hunk 2]
```diff
@@ -21,10 +21,33 @@ When you develop with NativeScript, you can choose between a local tool set ([Na
...
 > **TIP:** If you are familiar with the Apache Cordova CLI, you might want to try the NativeScript CLI. It provides a similar set of commands and an identical experience.
 
-With the open-source [NativeScript Command-Line Interface][NativeScript CLI] and an IDE or text editor of your choice, you can create, develop, store and build your apps entirely locally, free of charge, without an Internet connection and anonymously.
+With the open-source [NativeScript Command-Line Interface][NativeScript CLI] and an IDE or text editor of your choice, you can create, develop, store and build your apps entirely locally, free of charge and anonymously.
 
 To take advantage of this autonomy, you need to configure your system with the tools and SDKs for native development of the platforms for which you want to develop. The NativeScript CLI uses them to produce truly native builds of your cross-platform projects.
 
+### Quick setup
+
+If this is your first time developing mobile project, consider using our one-liner scripts to quickly setup your machine.
```
Comment (ErjanGavalji) [inline start/quick-setup.md]: When reading, one asks "what are these one-liner scripts?" and gets confused. We might better say "...consider using the one-liner scripts in this section to...".

Let's refrain from using "our". This is an open-source project and is owned by the entire world shall they want to contribute for good.

File: start/quick-setup.md [hunk 3]
```diff
@@ -21,10 +21,33 @@ When you develop with NativeScript, you can choose between a local tool set ([Na
...
+If you have experience developing mobile apps, you may skip to Advanced setup below.
```
Comment (ErjanGavalji) [inline start/quick-setup.md]: won't '...you may skip to the "Advanced Setup" section below.' be better?

File: start/quick-setup.md (hunk 2 above)
Comment (teobugslayer) [inline start/quick-setup.md]: I've used "our" in the sense of "provided by the project maintainers". At any rate, I will change the wording to more neutral one.

Comment (tjvantoll) [inline start/quick-setup.md]: developing mobile project --> developing _a_ mobile project

File: start/quick-setup.md [hunk 4]
```diff
@@ -21,10 +21,33 @@ When you develop with NativeScript, you can choose between a local tool set ([Na
 
 > **TIP:** If you are familiar with the Apache Cordova CLI, you might want to try the NativeScript CLI. It provides a similar set of commands and an identical experience.
 
-With the open-source [NativeScript Command-Line Interface][NativeScript CLI] and an IDE or text editor of your choice, you can create, develop, store and build your apps entirely locally, free of charge, without an Internet connection and anonymously.
+With the open-source [NativeScript Command-Line Interface][NativeScript CLI] and an IDE or text editor of your choice, you can create, develop, store and build your apps entirely locally, free of charge and anonymously.
```
Comment (tjvantoll) [inline start/quick-setup.md:24]: Just as an fyi this wording is duplicated on the Windows / OS X / Linux pages too.

**Comment by tjvantoll** [comment]
Just to make one suggestion: IMO it makes more sense for these scripts to be listed on the OS-specific pages—i.e. http://docs.nativescript.org/start/ns-setup-win and http://docs.nativescript.org/start/ns-setup-os-x—as there are a few places we link to the Windows/OS X specific pages already (for instance the Getting Started Guide). If we are going to keep the scripts listed on the quick setup, I would rename the OS-specific pages as the “advanced” guides.

**Comment by tjvantoll** [comment]
I’d also recommend telling people that these scripts may potentially take a really long time to complete to set the appropriate expectations.

**Comment by ErjanGavalji** [comment]
:+1: for @tjvantoll's links-from-os-specific-docs suggestion

File: start/quick-setup.md (hunk 4 above)
Comment (teobugslayer) [inline start/quick-setup.md:24]: Reworded there too, thanks for the tip

File: start/quick-setup.md (hunk 3 above)
Comment (teobugslayer) [inline start/quick-setup.md]: converted to a link to avoid any confusion.

**Comment by teobugslayer** [comment]
ping @tjvantoll @ikoevska @ErjanGavalji for final approval.

File: start/ns-setup-os-x.md [hunk 5]
```diff
@@ -32,6 +32,17 @@ On OS X systems, you can use the NativeScript CLI to develop Android and iOS app
     * Android SDK Build-tools 22.0.0 or a later stable official release
     * (Optional) Genymotion to expand your testing options
 
+### Quick setup
```
Comment (tjvantoll) [inline start/ns-setup-os-x.md:35]: Super nit picky, but this should probably be an h2 and not an h3.

File: start/ns-setup-os-x.md [hunk 6]
```diff
@@ -32,6 +32,17 @@ On OS X systems, you can use the NativeScript CLI to develop Android and iOS app
...
+
+If this is your first time developing a mobile project, consider using the one-liner scripts in this section to effortlessly setup your machine. 
+If you have experience developing mobile apps, you may skip to the [Environment Requirements](#environment-requirements) section below.
+
+Using Spotlight, search `Terminal` and start it. This opens a console window. Copy and paste this script:
```
Comment (tjvantoll) [inline start/ns-setup-os-x.md:40]: search for

File: start/quick-setup.md [hunk 7]
```diff
@@ -21,10 +21,33 @@ When you develop with NativeScript, you can choose between a local tool set ([Na
...
+If this is your first time developing a mobile project, consider using the one-liner scripts in this section to effortlessly setup your machine. 
+If you have experience developing mobile apps, you may skip to the [advanced setup](#advanced-setup) section below.
+
+ * On Windows:
+
+Open Start Menu, search for `Command Prompt` and start it. This opens a console window. Copy and paste this script:
+
+> @powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+
+You may need to accept an User Account Control prompt to grant the script administrative privileges. Note that the script downloads and installs some big dependencies and may take some time to complete.
```
Comment (tjvantoll) [inline start/quick-setup.md:39]: _a_ User Account not _an_ User account, because English has exceptions for everything.

Comment (tjvantoll) [inline start/quick-setup.md:39]: Same comment applies above obviously since this text is duplicated.

File: start/ns-setup-os-x.md (hunk 5 above)
Comment (tjvantoll) [inline start/ns-setup-os-x.md:35]: So after looking this over more, “Quick setup” should be an h2 on the OS X and Windows pages, but an h3 on the installation page.

**Comment by tjvantoll** [comment]
I just added a few small things. Overall this looks great.

The only concern I have is that the scripts themselves are now duplicated in multiple locations. The docs support Markdown snippets via `!!include` ([example](https://raw.githubusercontent.com/NativeScript/docs/production/start/getting-started.md)), so you may want to consider using that here so these scripts don’t get out of sync.

**Comment by teobugslayer** [comment]
@tjvantoll, i have fixed the typos. I'd rather leave to you to change the markdown to use includes because I do not feel confident enough using them. I agree that they are the better approach.

**Pull request merged**